cluster/gce/gci/configure-helper.sh

```diff
@@ -55,6 +55,27 @@ function config-ip-firewall {
   if [[ "${ENABLE_METADATA_CONCEALMENT:-}" == "true" ]]; then
     iptables -A KUBE-METADATA-SERVER -j DROP
   fi
+
+  # Flush iptables nat table
+  iptables -t nat -F || true
+
+  if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" && "${KUBERNETES_MASTER:-}" == false ]]; then
+    echo "Add rules for ip masquerade"
+    iptables -t nat -N IP-MASQ-AGENT
```

Comment:
I would call this something else - maybe just IP-MASQ  that way we know it was actually the ip-masq-agent that set this.

Reply:
Sure. Yeah this is not agent any more. Will do.

Reply:
Done